### PR TITLE
Support Multi User Feedback

### LINKED ISSUE #40: Milestone: Support multiple labelers
We currently assume that only a single feedback client is active at a time. We should support multiple labelers. For this, we need to consider how to assign queries to labelers, how to handle non-responses, etc.

### PR DESCRIPTION
**Pull request by patrickab**
By maintaining a second level datastructure (implemented as a list), for sent, yet unevaluated queries we can now support multiple labelers.

Non-responded queries are simply be taken out of the list, whenever no queries remain in the queue. To do so, we simply select the oldest unevaluated query `query_list[0]`, remove it from the first position and put it to the last position of the list. This procedure ensures, that we always re-send the oldest unevaluated query.

Closes #40

### CODE REVIEW

File: prefq/server.py [hunk 1]
```diff
@@ -47,7 +47,8 @@
 app.config["VIDEO_FOLDER"] = "videos"
 
 feedback_data = {}
-pending_queries = queue.Queue()
+query_queue = queue.Queue()
+query_list = []
```
Comment (timokau) [inline prefq/server.py:51]: Please give the two variables names that indicate what they are intended to store (something like `queries_not_yet_served` and `queries_pending_response`).

**Review by timokau — CHANGES_REQUESTED** [review]
What happens if we serve a query to two users (because one user was very slow to respond) and get two responses eventually?

File: prefq/server.py (hunk 1 above)
Comment (patrickab) [reply prefq/server.py]: done!

**Review by timokau — APPROVED** [review]
Thanks!

**Pull request merged**